sklearn/covariance/_robust_covariance.py

```diff
@@ -698,6 +699,14 @@ class MinCovDet(EmpiricalCovariance):
     array([0.0813... , 0.0427...])
     """
 
+    _parameter_constraints = {
+        **EmpiricalCovariance._parameter_constraints,  # type: ignore
+        "support_fraction": [
+            Interval(Real, 0, 1, closed="neither"),
```

Comment:
Uhm I think that we should be supporting `1.0` here.

```suggestion
            Interval(Real, 0, 1, closed="right"),
```

Reply:
done

Reply:
The docstring says otherwise. Let's correct it as well

Reply:
updated `support_fraction` description in `MinCovDet` docstring:
`(0,1) -> (0,1]`